__tests__/AppCatalog.js

```diff
@@ -231,7 +231,14 @@ describe('Apps and App Catalog', () => {
         message: `We're installing your app called 'test-app' on ${V4_CLUSTER.id}`,
       };
       nock(API_ENDPOINT)
-        .intercept(`/v4/clusters/${V4_CLUSTER.id}/apps/test-app/`, 'PUT')
+        .intercept(`/v4/clusters/${V4_CLUSTER.id}/apps/test-app/`, 'PUT', {
```

Comment:
This is a regression test for what happened. It makes sure that the request that Happa makes includes the expected JSON body